src/kafka.rs

```diff
@@ -93,3 +94,13 @@ fn pathbuf_to_string(path: &PathBuf) -> crate::Result<&str> {
     path.to_str()
         .ok_or_else(|| KafkaError::InvalidPath { path: path.into() }.into())
 }
+
+pub(crate) struct KafkaStatisticsContext;
+
+impl ClientContext for KafkaStatisticsContext {
+    fn stats(&self, statistics: Statistics) {
+        emit!(KafkaStatisticsReceived::new(statistics));
```

Comment:
Super minor nitpick, but we tend not to use constructors for trivial cases like this. That'll actually make this callsite a little shorter and we can drop the struct's `impl` block entirely.